### PR TITLE
update k8s configs and add example grafana dashboard

### PR DESCRIPTION
**Pull request by stfung77**
- update k8s example configs
- add example grafana dashboard

### CODE REVIEW

File: docs/resources/deployment-examples/example-k8s/anchor-platform-dev/reference-server-deployment.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
       - name: reference-server
-        image: docker-registry.services.stellar-ops.com/dev/anchor-platform:98e5ae9
```
Comment (erika-sdf) [inline docs/resources/deployment-examples/example-k8s/anchor-platform-dev/reference-server-deployment.yaml:23]: I'm missing context on this; why are we moving this to dockerhub?

File: docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md [hunk 2]
```diff
@@ -193,7 +196,7 @@ The format of anchor platform's logs can be set by the `LOG_APPENDER` environmen
     "time": timestamp,
     "source": logger,
     "index": location,
-    "event" {
+    "event": {
```
Comment (erika-sdf) [inline docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md:199]: Thanks for catching this

**Review by erika-sdf — APPROVED** [review]
Looks good, just have one question.

File: docs/resources/deployment-examples/example-k8s/anchor-platform-dev/reference-server-deployment.yaml (hunk 1 above)
Comment (stfung77) [reply docs/resources/deployment-examples/example-k8s/anchor-platform-dev/reference-server-deployment.yaml:23]: dockerhub is the official repo. this docker-registry one is used by our CI/CD pipeline

**Pull request merged**